app/utils/jspsych/functions.js

```diff
@@ -123,11 +140,20 @@ export const instantiateTimeline = (
         if (trial.id === 'trial') {
           return {
             ...trial,
-            on_start: () =>
+            on_start: e => {
+              if (e) {
+                const correctResponse = parseFloat(e.correct_response);
```

Comment:
for the on_start, is `e` the callback event? I think that we would like to have the eventType passed as the marker to the LSL string similar to what the `eventCallback` is doing below.

Reply:
Thanks, I see the idea now. In the new version, eventType is passed as a float number. I have not found support for string in node-lsl. Shall I work on adding it to the node-lsl library, or leave like this with float numbers?

Reply:
I think leaving it to just float numbers should be fine